src/main/java/duke/Parser.java

```diff
@@ -29,12 +34,24 @@ public static Command stringToCommand(String command) {
         }
     }
 
-    public static LocalDate stringToDate(String duration) {
-        LocalDate localDate = LocalDate.parse(duration);
-        return localDate;
+    public static String[] deadlineSplit(String deadline) {
+        return deadline.split(" ", 2);
+    }
+
+    public static String[] eventSplit(String event) {
+        return event.split(" ", 6);
+    }
+
+    public static LocalDateTime stringToDateTime(String duration) {
+        LocalDateTime localDateTime = LocalDateTime.parse(duration, strFormatter);
+        return localDateTime;
+    }
+
+    public static String dateTimeToString(LocalDateTime localDateTime) {
```

Comment:
Should methods such as `deadlineSplit` be named something like `splitDeadlineIntoStringArray` to better follow the convention of naming methods using verbs?